exercise/exercise_models.py

```diff
@@ -575,7 +576,15 @@ def _check_submission_allowed(self, profile, request=None):
 
         # Support group id from post or currently selected group.
         group = None
-        group_id = request.POST.get("_aplus_group") if request else None
+        group_id = None
+        if request:
+            try:
+                group_id = json.loads(request.POST.get('__aplus__','{}')).get('group')
```

Comment:
This was said earlier too. There should be a space between function parameters: `.get('__aplus__', '{}')`